### ISSUE TITLE
Error message persists and hides running query

### ISSUE THREAD

**Issue by MikeB2019x** [dbeaver/dbeaver#17178]
#### System information: 
- MacOS Version 10.15.7
- DBeaver Version 22.1.1.202206261508

#### Describe the problem you're observing:
This is my screen on the second run of the query.  
![image](https://user-images.githubusercontent.com/49003263/181096723-6386bf85-79a3-4ae9-8c36-f610b322f98c.png)

There are two things wrong:

1. the error was thrown when trying to filter the results in the tab (but the columns _do_ exist ... problem for another day); and 
2. when the error was thrown the tab results showed the error message in the results tab.  So I relaunched the query and what you see in the image is a query being run.  Note that there is no indication the query is running ie. no progress bar, etc.  Eventually (because this query takes about 4 min) the results are displayed.

#### Steps to reproduce, if exist:
1) create a long running query with something that will trigger an error;
2) correct the error and relaunch the query in the same tab.  The error will continue to be displayed with no indication that your query is running.

#### Expected Behaviour
If I rerun a query in the same tab I would expect:

1. the results from last query attempt to clear incl. error messages; and
2. I would expect to see the progress bar indicating that the query is in progress

What would be nice:

1. I run a query and get some results.
2. I try to filter _in the tab_ and if there's an error, an error indicator shows up in the bar below the results (like it used to) BUT the results from my previous query _aren't_ replaced/overwritten.  Instead I could still see the results of the last successful query before I tried filtering in the tab and the error indicator (that I can click on to expand).  Only a new successful query will change the tab results.

What would be really nice:
Return the UI to where the error is indicated beneath the results and can be expanded if needed.

**Comment by uslss** [comment]
@MikeB2019x you can enable Show errors in a dialog in Preferences. Does it resolve your issue?
![image](https://user-images.githubusercontent.com/31996417/181504584-60c0dc88-eb95-4370-8ef6-31cec8ca8a62.png)

**Comment by MikeB2019x** [comment]
@uslss yup that falls in to the 'nice' category so many thanks for that.  _Really nice_ would be being able to prevent even the pop-up dialog b/c when I'm in the field on a small screen it's just one more thing blocking my view :)

**Comment by dbeaver-pm** [comment]
Thanks for the idea @MikeB2019x
